lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -81,7 +83,7 @@ def test_clean_emptys(self, server_config, make_logger):
 
     def test_create_bad(self, monkeypatch, server_config, make_logger, tarball):
         # Calling restorecon() gives warning messages about "no default label"
-        monkeypatch.setattr("selinux.restorecon", lambda a: None)
+        monkeypatch.setattr("selinux.is_selinux_enabled", lambda a: False)
```

Comment:
The "lifecycle" test doesn't mock out `is_selinux_enabled` to "false" (actually, the real one returns integer 0 or 1), but I didn't add a mock to make sure it reported `1`, either, so I'll do that.

Reply:
Yeah, I only said "True" because your mock here returned `False`.  🙂 

It would probably be good to have a test for operation under each case.

Regardless, knowing now what I know about `is_selinux_enabled()`, we should probably be mocking it regardless, just so that we know what the environment is that the test is running in.  (That is, we don't want the test to behave differently depending on what the host environment just happens to be....)

Reply:
I'm making this a bit more clear by creating `selinux_enabled` and `selinux_disabled` fixtures to encapsulate the states. (And you're right that `is_selinux_enabled()` doesn't take a parameter: I was thinking of it somehow as an instance method with an implicit `self` when I did that, but I don't think it is.)

It occurs to me that a "one+" would be to track calls and detect whether calls had been made or not; I think there ought to be ways within `pytest` to track that state a bit more cleanly than the other mock tracking I did earlier, but I'm not getting into that today.